specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-30/disk.json

```diff
@@ -2089,6 +2119,11 @@
           "type": "integer",
           "format": "int64",
           "description": "If createOption is Upload, this is the size of the contents of the upload including the VHD footer. This value should be between 20972032 (20 MiB + 512 bytes for the VHD footer) and 35183298347520 bytes (32 TiB + 512 bytes for the VHD footer)."
+        },
+        "logicalSectorSize": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Support customers creating ultra disks with logicalSectorSize 512E. Support values are 512 and 4096 (default value)."
```

Comment:
Logical sector size for Ultra disks. Supported values are 512 and 4096. Default value is 4096.